csharp/ql/src/utils/modelgenerator/internal/CaptureSinkModelsQuery.qll

```diff
@@ -0,0 +1,18 @@
+private import semmle.code.csharp.dataflow.ExternalFlow
+private import semmle.code.csharp.dataflow.TaintTracking
+private import CaptureModels
+
+module PropagateToSink = TaintTracking::Make<PropagateToSinkConfig>;
```

Comment:
I think I'd favour moving these module instantiations to be next to the configurations. It's completely safe to put multiple such instantiations in scope - unlike the previous configuration classes.

Reply:
Will that cause a performance issue (If one only wants to execute one of the queries then all configuration modules will still be created)? There are three different queries that relies in the content of CaptureModels.qll.

Reply:
> Will that cause a performance issue
No.